index.js

```diff
@@ -31,44 +32,82 @@ const preamble = [
 	"We'll never #resist @realdonaldtrump without",
 	"what's the best way to #disrupt #startups? Simple:",
 	"When will @elonmusk have",
+	"Have people tried putting",
+	"Not enough companies have",
+	"Thought leaders are trying to get",
+	"Crossing the chasm requires",
+	"All top scale-ups know you need",
+	"Serious talk: I need to see",
+	"In 2021, the world needs",
+	"It's 2020! When will we get",
 ];
 
 const number = [
-	"10X",
+	"10X more",
 	"a few",
-	"a fucking butt tonne",
-	"a handful",
-	"juuuuuusst a few",
-	"lots",
+	"a fucking butt tonne more",
+	"a handful of",
+	"juuuuuusst a few more",
+	"lots of",
 	"many",
 	"many many",
 	"many many many",
 	"many many many many",
-	"oodles",
+	"oodles of",
 	"some",
-	"tonnes",
+	"tonnes of",
+	"more",
+	"a board room full of",
+	"a slide deck of",
+	"Unsplashed stock photos with",
+	"enough",
+	"the",
+	"a team of",
 ];
 
 const xx = [
-	"broards",
-	"chiquitas",
-	"cuties",
-	"dames",
-	"females",
-	"gurls",
-	"honey bunnies",
-	"hoochie coochie mammas",
-	"laaadies",
-	"ladies",
-	"lovely ladies",
-	"lovely lovely ladies",
-	"lovely lovely lovely ladies",
-	"lovely lovely lovely lovely ladies",
-	"of the fairer sex",
-	"senioritas",
-	"skirts",
-	"spicy mamma jammas",
-	"womenfolk",
+	"behind keyboards",
```

Comment:
I think the removal of the sexism may be a bug not a feature, in this case

Reply:
fixed